.devcontainer/devcontainer.json

```diff
@@ -2,19 +2,18 @@
 
 {
 	"name": "Go",
-	"build": {
-		"dockerfile": "Dockerfile",
-		"args": {
-			"VARIANT": "1.17-bullseye",
-		}
-	},
 	"containerEnv": {
-		"GO_OPENSSL_VERSION_OVERRIDE": "1.1.0",
+		"GO_OPENSSL_VERSION_OVERRIDE": "1.1.1"
 	},
-	"onCreateCommand": "sh ${containerWorkspaceFolder}/scripts/openssl.sh ${GO_OPENSSL_VERSION_OVERRIDE}",
+	"features": {
+		"ghcr.io/devcontainers/features/go:1": {}
+	},
+	"onCreateCommand": "sudo sh ${containerWorkspaceFolder}/scripts/openssl.sh ${GO_OPENSSL_VERSION_OVERRIDE}",
 	"customizations": {
 		"vscode": {
-			"extensions": ["golang.go"]
+			"extensions": [
+				"golang.go"
```

Comment:
I would expect that this is included by default based on https://github.com/devcontainers/features/tree/main/src/go#customizations and the fact that it's a pretty standard extension, but I wasn't able to find a clear answer whether these are *available* customizations to pick from or *default* customizations. 😄 Do you know for sure if this bit is required?

Reply:
Hmm, for what I could test, looks like it's not necessary. The only properties settable by users are the ones under `options`. In fact there is a proposal to allow opting out to the default extensions: https://github.com/devcontainers/features/issues/386.